pkg/operator/controllers/guardrails/policies/gktemplates-src/aro-regularuser-validation/src_test.rego

```diff
@@ -0,0 +1,91 @@
+package aroregularuservalidation
+
+import data.aroregularuservalidation
+
+test_denies_regular_user_access {
+    input := {
```

Comment:
I think you can create a function for generating inputs instead of creating input obj in each cases